vita3k/gui/src/themes.cpp

```diff
@@ -249,6 +255,10 @@ bool init_theme(GuiState &gui, EmuEnvState &emuenv, const std::string &content_i
                 if (!home_property.child("m_hostCollabo").child("m_iconFilePath").text().empty())
                     theme_icon_name["NPXS10026"] = home_property.child("m_hostCollabo").child("m_iconFilePath").text().as_string();
 
+                // Bgm theme
+                if (!home_property.child("m_bgmFilePath").text().empty())
+                    path_bgm = { "ux0", (fs::path("theme") / content_id / home_property.child("m_bgmFilePath").text().as_string()).string() };
```

Comment:
```suggestion
                    path_bgm = { "ux0", fmt::format("theme/{}/{}",content_id,home_property.child("m_bgmFilePath").text().as_string()) };
```
No need to do double conversion string->path->concatenation->string